google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITResultSetGetValue.java

```diff
@@ -429,6 +522,59 @@ public void testReadNullValuesInArrays() {
     }
   }
 
+  @Test
+  public void testReadNullValuesInArraysPostgreSQL() {
```

Comment:
Why do we need separate tests for GoogleSQL and PostgreSQL in this case? They seem to be equal. Or am I missing something?

Reply:
structs and json are not supported and column names are different as PG requires explicit quotes for case